app/Http/Controllers/MinesController.php

```
@@ -0,0 +1,273 @@
+
+use App\Http\Resources\MineResource;
+use App\Http\Services\BalanceService;
+use App\Models\MineGame;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+use InvalidArgumentException;
+
+class MinesController
+{
+
+    public function current(Request $request){
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)